app/views/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is a list of posts for a given user</h1>
```

Comment:
- [ ] Please add a `show` file that corresponds to the show method in the post_controller